docs/en_US/install/3ds.md

```diff
@@ -1,23 +1,24 @@
 # 3DS/2DS Family
 
-<div class="tip red">
-	<strong>CAUTION:</strong>
-	SYSTEM TRANSFERS ARE NOT CURRENTLY SUPPORTED BY OUR SERVERS. ATTEMPTING TO PERFORM A SYSTEM TRANSFER MAY PREVENT YOU FROM BEING ABLE TO GO ONLINE IN THE FUTURE. SUPPORT FOR SYSTEM TRANSFERS IS IN DEVELOPMENT.
-</div>
+## Things to know before you start
 
-<div class="tip red">
-	<strong>CAUTION:</strong>
-	Collecting badges in Nintendo Badge Arcade while connected to one network and then launching the game on a different network will result in your badges disappearing. This occurs because the locally saved data does not match the data stored on the server.
-</div>
+This guide assumes that you have a **Homebrewed System** running the latest version of Luma3DS (13+). If you are unsure or do not have a homebrewed system, please follow the [3DS Hacks Guide](https://3ds.hacks.guide/) to check, install and update it. 
```

Comment:
I'm not sure I agree with removing the caution callouts. This was done intentionally to be eye grabbing and bring attention to the information inside. These are pretty important things to know about, and having them just look like the rest of the page makes it more likely someone will skim over them

@hauntii @gitlimes @DaniElectra thoughts?

Reply:
Yeah I disagree with inlining the cautions and tips, I think how they are rn it's already fine

Reply:
I would argue that having 2 back-to-back callouts in **the beginning** of the guide is almost too much. A person who just wants to get Pretendo installed might ignore the callouts and skip straight to the body of the guide.

I think that having a "Prerequisites" section would be better as it is the first thing that they will read anyway.

Reply:
Having callouts for important information at the start of guides/wikis is fairly standard practice. You can see this on wikis like Wikipedia and even the official homebrew guide. It's done this way for a reason, to be immediately eye catching to the user. Putting key information like this as plain text like anything else definitely runs more of a risk of people missing it. Yes, people can still skip callouts. But it's far less likely

![Screenshot from 2024-04-24 11-09-07](https://github.com/PretendoNetwork/website/assets/27011796/67bfdcf7-e256-48a9-97a5-eb0a560572e8)
![Screenshot from 2024-04-24 11-09-25](https://github.com/PretendoNetwork/website/assets/27011796/422266bc-32b8-404a-9709-34c7ef391218)